script/generate_views

```diff
@@ -153,11 +153,17 @@ def create_views_if_not_exist(client, views, exclude, sql_dir):
                 " AS metadata"
             ]
             # Add special replacements for Glean pings.
-            if table.labels.get("schema_id", None) == "glean_ping_1":
+            schema_id = table.labels.get("schema_id", None)
+            if schema_id == "glean_ping_1":
                 replacements += [
                     "`moz-fx-data-shared-prod.udf.normalize_glean_ping_info`(ping_info)"
                     " AS ping_info"
                 ]
+            elif schema_id in ("main_ping_1", "main_ping_4"):
```

Comment:
Once https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/479 makes it to prod, the schema_id will have a 4 rather than a 1.